README.md

```diff
@@ -27,3 +27,13 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+## Aijun Hall
+
+An invariant for my recursive implementation of `fib()` could be that each new `fib(n)` is the correctly calculated sum value of the two immediately preceding values, `fib(n-1)` and `fib(n-2)`. To confirm every computed value of `fib(n)` is correct, we also rely on the fact that every element in `prev_fib_array` is derived by adding the two previous elements.
```

Comment:
But this is not true before the current iteration, i.e. before the current Fibonacci value is computed.